src/jsutils.h

```diff
@@ -57,21 +60,30 @@ size_t flash_strlen(const char *str);
 /// Copy a string from flash to RAM
 char *flash_strncpy(char *dest, const char *source, size_t cap);
 
+/// Compare a string in memory with a string in flash
+int flash_strcmp(const char *mem, const char *flash);
```

Comment:
It would be useful to have the flash_strcmp, flash_strcpy and flash_strlen functions available globally, so that other optimisations can be made.

Would using `#define strcmp(x,y)   flash_strcmp(x,y)` make the need for `#ifdef` conditionals reduce, so that all strcmp are using this code?

Then there would be no need for special versions of `jsvNewFromString` as they would just *work*